prime-router/metadata/fhir_mapping/hl7/codesystem/ExtensionUrlMapping.yml

```diff
@@ -315,13 +279,10 @@
   url: "https://reportstream.cdc.gov/fhir/StructureDefinition/xcn-practitioner"
 
 # NK1 -> RelatedPerson
-# TODO: collapse
-- id: "nk1-13-organization-name"
-  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/nk1-13-organization-name"
+- id: "nk1-related-person"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/nk1-related-person"
 - id: "nk1-15-administrative-sex"
```

Comment:
I could see us going either way here, it's fine to leave it as an extension on the `gender`, but it might be cleaner to take that off and just put it into the overall extension like everything else, leaving the `gender` without an extension just for conformance to the mapping. Don't feel strongly either way, this just looks a bit like an outlier, so curious what you think.